src/igvfd/schemas/file.json

```diff
@@ -0,0 +1,151 @@
+{
+    "title": "File",
+    "$id": "/profiles/file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Abstract schema for submission of a file.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "md5sum",
+        "file_format",
+        "content_type"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        }
+    ],
+    "dependentSchemas": {
+        "validation_error_detail": {
+            "comment": "Validation Error Detail can only be specified on Files that did not pass validation.",
+            "required": [
+                "upload_status"
+            ],
+            "properties": {
+                "upload_status": {
+                    "enum": [
+                        "invalidated"
+                    ]
+                }
+            }
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "hidden comment": "Bump the default in the subclasses."
+        },
+        "accession": {
+            "accessionType": "FF"
+        },
+        "content_md5sum": {
+            "title": "Content MD5sum",
+            "description": "The MD5sum of the uncompressed file.",
+            "type": "string",
+            "permission": "import_items",
+            "format": "hex",
+            "maxLength": 32,
+            "pattern": "[a-f\\d]{32}|[A-F\\d]{32}"
+        },
+        "content_type": {
+            "title": "Content Type",
+            "description": "The type of content in the file.",
+            "comment": "This property is overwritten by the subclasses to define specific enum values.",
+            "type": "string",
+            "enum": []
+        },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External Resources",
+            "description": "Identifiers from external resources that may have 1-to-1 or 1-to-many relationships with IGVF objects.",
```

Comment:
I think it should be specific to "IGVF file objects" rather than general "IGVF objects"